tests/testing.py

```diff
@@ -1267,3 +1268,31 @@ def f(u, v, arg, kwarg=-1.0):
 
     assert 0.0 == pytest.approx(np.linalg.norm(coordinates - coordinates_ref), 1e-12)
     assert np.linalg.norm(connectivity - connectivity_ref) == 0
+
+
+def test_import_fluent_geometry():
+    """ 
+    Test if the simplest comand to import a fluent geometry works without error. 
+    """
+    
+    fluent_geometry= os.path.join(testing_input_geometry,"fluent_aneurysm.msh")
+    cubit = CubitPy()
+    cubit.Import_Fluent_Geometry(fluent_geometry)
+    assert False == cubit.was_last_cmd_undoable()
```

Comment:
I didn't know that the command `was_last_cmd_undoable` exists. Why do you check this here? Is this an error indicator? If so, we should maybe add the check directly to the cubit interface whenever a command is passed on to cubit?

Reply:
yes, if for a reason (wrong syntax, not readable input file, etc... ) the cubit command can not be executed and/or fails, the error is indicated with this. 
With this check, I simply test if the importation process was at least successful.

I'm not quite sure if we should add it every time. Essentially, we will end up checking things double. We could try it first out on the test suite and some examples and see if something fails. But for cubit commands
 which tend to fail(meshing scheme), it's definitely a nice command to get feedback!

Reply:
You are right, a check every time is probably not wat we want. But good to know that such a check exists!